Tests/POEApi.Model.Tests/PoeModelTests.cs

```diff
@@ -205,6 +206,36 @@ public void GetShardCurrencyStashTest()
             }
         }
 
+        [TestMethod]
+        public void GetFragmentStashTest()
+        {
+            string fakeStashInfo = Encoding.UTF8.GetString(Files.SampleFragmentStash);
+            using (var stream = GenerateStreamFromString(fakeStashInfo))
+            {
+                _mockTransport.Setup(m => m.GetStash(0, "", "", false)).Returns(stream);
+
+                var stash = _model.GetStash(0, "", "");
+
+                Assert.IsNotNull(stash);
+
+                Assert.AreEqual(1, stash.Tabs.Count);
+
+                var items = stash.GetItemsByTab(12);
+
+                
+                var viewModel = new FragmentStashViewModel(items); 
+
+                Assert.AreEqual(4, viewModel.ChayulaSplinter.Item.StackSize);
+                Assert.AreEqual(6, viewModel.Offering.Item.StackSize);
+                Assert.AreEqual(11, viewModel.Dawn.Item.StackSize);
+                Assert.AreEqual(8, viewModel.TulSplinter.Item.StackSize);
+                Assert.AreEqual(11, viewModel.XophSplinter.Item.StackSize);
+                Assert.AreEqual(17, viewModel.EshSplinter.Item.StackSize);
+                Assert.AreEqual(4, viewModel.ChayulaSplinter.Item.StackSize);
+                Assert.AreEqual(1, viewModel.DivineVessel.Item.StackSize);
```

Comment:
Include a check for at least one of the items whose `StackSize` should be 0.

Reply:
There won't be an item there so it won't have a stack. I'll assert that it is null instead.